app/controllers/application_controller.rb

```diff
@@ -1,2 +1,19 @@
 class ApplicationController < ActionController::Base
+
+  helper_method :current_user, :logged_in?
+
+  def current_user
+    @current_user ||= User.find(session[:user_id]) if session[:user_id]
+  end
+
+  def logged_in?
+    !!current_user
```

Comment:
double `!!`

Reply:
it's supposed to be double

Reply:
ok, but why?

Reply:
it's converting into proper boolean value